### DISCUSSION TITLE
Informatii Nod pentru incepatori

### DISCUSSION THREAD

**Discussion by d4dane** [CoopNetwork/GMD-blockchain#18] [Q&A]
Buna ziua, fiind incepator in domeniu am nevoie de cateva lamuriri. Am reusit ieri sa fac un nod pe unbuntu, este online, ma intereeaza ce am de facut mai departe pentru a obtine Hallmark si a genera cei 15% apy pentru nod. Ma intereseaza care sunt pasii concreti: trebuie sa cer Hallmark si daca da cum, unde atasez walletul GMD, dobanda de nod se primeste anual pentru tot GMD-ul din wallet (se fac snapshots?), cum vad daca nodul este activ si genereaza apy, care este suma minima de GMD tot timpul pentru ca nodul sa genereze apy? De asemenea ce este mai rentabil, sa dau GMD-ul in leasing sau sa fac forging? Sunt multe intrebari, sper ca raspunsurile sa ajute si alte persoane. Multumesc!

**Comment by GeomaDAO** [comment]
Buna,
Scuze de răspunsul întârziat.Este excelent ca ți-ai făcut nod. Detalii mai în amănunt o sa le postam pe GitHub ,probabil sept.viitoare.O sa postam anunțul pe grup.
Multumesc ca participi la descentralizare.


Pe curând,

Arthur
